esp/esp/program/modules/handlers/teachercheckinmodule.py

```diff
@@ -167,7 +167,9 @@ def ajaxteachertext(self, request, tl, one, two, module, extra, prog):
             if 'username' in request.POST and 'section' in request.POST:
                 sec = ClassSection.objects.get(id=request.POST['section'])
                 teacher = PersistentQueryFilter.create_from_Q(ESPUser, Q(username=request.POST['username']))
-                message = "Don't forget to check-in for your " + one + " class that is scheduled for " + sec.friendly_times(include_date = True)[0] + "!"
+                template = get_template(self.baseDir() + 'teachertext.txt')
+                context = {'prog': prog, 'one': one, 'two': two, 'sec': sec, 'teacher': teacher}
```

Comment:
If you read the template from a file in the codebase, how will chapters customize it?  If you read it from a tag instead, it would be easier for chapters to customize. Since this is a text message, it will be relatively short so putting the entire thing in a tag is reasonable.

Reply:
Oh, I just read the original issue #2734 and realized that they could use a template override.